app/models/concerns/avatarable.rb

```diff
@@ -6,5 +6,15 @@ module Avatarable
 
     validates :avatar, content_type: ["image/png", "image/jpeg", "image/jpg"],
       max_file_size: 2.megabytes
+
+    validate :avatar_validation, on: :create
```

Comment:
```suggestion
    validates :avatar, presence: true, on: :create
```

Does this work?

Reply:
No but i'm working on a fix. Learning about custom validators. I realized just now that max_file_size and content_type were custom and are in the validators folder lol

Reply:
Nice! Looking forward to it. @ me in a comment when it's ready for review again :)

Reply:
@joemasilotti check out the new code. 

new failed test - the cause :
The following data requires normalization:
Run `i18n-tasks normalize` to fix
config/locales/en.yml

Should I proceed and normalize the data?

Reply:
Yes please! That will only touch the locale file.

Reply:
@joemasilotti done

Reply:
new errors below :

Error:
DeveloperTest#test_successful_profile_creation_sends_a_notification_to_the_admins:
ActiveRecord::RecordInvalid: Validation failed: Avatar is not attached, Search status is not included in the list, Time zone can't be blank
    test/models/developer_test.rb:105:in `block (2 levels) in <class:DeveloperTest>'
    test/models/developer_test.rb:104:in `block in <class:DeveloperTest>'

Reply:
The params you are sending in the test need to now also include the ones you made non-optional.

Reply:
right, time to learn again :) Let me get on that. Thanks again for the patience

Reply:
just to specify, having trouble attaching the avatar on the test